### PR TITLE
Various documentation improvements

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: packages/flutter/lib/src/services/system_channels.dart [hunk 1]
```diff
@@ -12,28 +12,159 @@ class SystemChannels {
...
   ///
-  /// Ignores missing plugins.
+  /// The following outgoing methods are defined for this channel (invoked using
+  /// [OptionalMethodChannel.invokeMethod]):
+  ///
+  ///  * `Clipboard.setData`: Places the data from the `text` entry of the
+  ///    argument, which must be a [Map], onto the system clipboard. See
+  ///    [Clipboard.setData].
+  ///
+  ///  * `Clipboard.getData`: Returns the data from the argument, a [String],
```
Comment (goderbauer) [inline packages/flutter/lib/src/services/system_channels.dart]: I don't understand what this sentence is supposed to tell me.

Comment (goderbauer) [reply packages/flutter/lib/src/services/system_channels.dart]: Should it maybe be "from the system clipboard"?

**Review by goderbauer — APPROVED** [review]
LGTM

I learnt a lot from reading the new doc strings. Thanks for adding them!

File: packages/flutter/lib/src/services/system_channels.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/services/system_channels.dart]: fixed.

**Comment by Hixie** [comment]
will land on green

**Pull request merged**